core/common/src/main/java/alluxio/exception/ExceptionMessage.java

```diff
@@ -129,6 +129,8 @@
   DELETE_FAILED_DIRECTORY_NOT_IN_SYNC(
       "Cannot delete {0} because the UFS has contents not loaded into Alluxio. Sync Alluxio with "
           + "UFS or run delete with unchecked flag to forcibly delete"),
+  DELETE_NONEMPTY_DIRECTORY_FAILED_CHILDREN(
+      "Cannot delete non-empty directory {0}. Failed to delete children: {1}"),
```

Comment:
Can we remove the "nonempty" part? It's implied by saying we failed to delete the children